admin_manual/installation/php_56_upgrade.rst

```diff
@@ -0,0 +1,149 @@
+
+You will first require to subscribe to the Red Hat Software Collections channel repository to download and install the PHP56 package in RHEL 7
+
+.. code-block:: console
+   
+   subscription-manager repos --enable rhel-server-rhscl-7-rpms
+
+Then proceed by installing the required PHP 5.6 modules
+
+.. code-block:: console
+
+   yum install rh-php56 rh-php56-php rh-php56-php-gd rh-php56-php-mbstring rh-php56-php-mysqlnd rh-php56-php-intl rh-php56-php-ldap
+
+Enable PHP 5.6 system-wide
+
+.. code-block:: console
+
+   cp /opt/rh/rh-php56/enable /etc/profile.d/rh-php56.sh source /opt/rh/rh-php56/enable
+
+Disable loading the old PHP 5.4 Apache modules by changing their names
+
+.. code-block:: console
+
+   mv /etc/httpd/conf.d/php.conf /etc/httpd/conf.d/php54.off
+   mv /etc/httpd/conf.modules.d/10-php.conf /etc/httpd/conf.modules.d/10-php54.off
+
+Copy the PHP 5.6 Apache modules into place
+
+.. code-block:: console
+
+   cp /opt/rh/httpd24/root/etc/httpd/conf.d/rh-php56-php.conf /etc/httpd/conf.d/
+   cp /opt/rh/httpd24/root/etc/httpd/conf.modules.d/10-rh-php56-php.conf /etc/httpd/conf.modules.d/
+   cp /opt/rh/httpd24/root/etc/httpd/modules/librh-php56-php5.so /etc/httpd/modules/
+
+Restart Apache
+
+.. code-block:: console
+
+   service httpd restart
+
+Upgrading to PHP 7.0
+--------------------
+
+You will first require to subscribe to the Red Hat Software Collections channel repository to download and install the PHP56 package in RHEL 7
+
+.. code-block:: console
+   
+   subscription-manager repos --enable rhel-server-rhscl-7-rpms
+
+Then proceed by installing the required PHP 5.6 modules
+
+.. code-block:: console
+
+   yum install rh-php70 rh-php70-php rh-php70-php-gd rh-php70-php-mbstring rh-php70-php-mysqlnd rh-php70-php-intl rh-php70-php-ldap
+
+Enable PHP 5.6 system-wide
+
+.. code-block:: console
+
+   cp /opt/rh/rh-php70/enable /etc/profile.d/rh-php70.sh source /opt/rh/rh-php70/enable
+
+Disable loading the old PHP 5.6 Apache modules by changing their names
+
+.. code-block:: console
+
+   mv /etc/httpd/conf.d/php.conf /etc/httpd/conf.d/php56.off
+   mv /etc/httpd/conf.modules.d/10-php.conf /etc/httpd/conf.modules.d/10-php56.off
+
+Copy the PHP 5.6 Apache modules into place
+
+.. code-block:: console
+
+   cp /opt/rh/httpd24/root/etc/httpd/conf.d/rh-php70-php.conf /etc/httpd/conf.d/
+   cp /opt/rh/httpd24/root/etc/httpd/conf.modules.d/15-rh-php70-php.conf /etc/httpd/conf.modules.d/
+   cp /opt/rh/httpd24/root/etc/httpd/modules/librh-php70-php7.so /etc/httpd/modules/
+
+Restart Apache
+
+.. code-block:: console
+
+   service httpd restart
+
+Installing Redis Support
+------------------------
+
+To add the Redis client extension, run the commands below. 
+
+.. code-block:: console
+   
+   yum install rh-php70-php-devel
+   pecl install redis 
+
+Then, when the installation add a configuration file to enable the extension, using the following commands.
```

Comment:
I mainly pushed it so that it wouldn't just be on my laptop (not to infer that I don't trust my hardware).